### ISSUE TITLE
sync with upstream v1.2.6

### ISSUE THREAD

**Issue by markybob** [rpwoodbu/mosh-chrome#125]
wishlist of syncing to upstream v1.2.6

**Comment by rpwoodbu** [comment]
Yep, saw the announcement, and plan to do this soon. I already have the RC in "Mosh (dev)", and will push v1.2.6 there soon (probably today). After it has baked there for a bit (a few days, probably), I'll promote dev to stable.

**Comment by rpwoodbu** [comment]
This is going out to "Most (dev)" now. I'll actually keep this issue open until it hits stable. Issues that folks think they might be having on "Mosh (dev)" due to this update should go here.

**Comment by binaryhermit** [comment]
Mosh dev version crashes on a chromebook flip, both on the beta channel and the dev channelm when logging into a server, either using public key or password, giving an error message about mosh NaCL crashing.  When using a password, the error message appears after entering the password.

**Comment by rpwoodbu** [comment]
Thanks for that report, @binaryhermit. I didn't bother to test this on ARM for the dev track. I am seeing the same crash on my HP 11. It seems that this old ARM Clang PNaCl bug still exists in Pepper 49, even though the [issue](https://bugs.chromium.org/p/nativeclient/issues/detail?id=4173) I opened about it long ago was marked "WontFix". 😞 I've just pinged that issue. I'm going to leave this broken in dev for the time being, in case someone actually responds to my ping and wants an easy repro.

This is fixed by simply re-translating the PNaCl bitcode for ARM. I'll be sure to do that and test it on ARM before I release to stable.
